src/components/Reservation.js

```diff
@@ -0,0 +1,106 @@
+import { React, useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { FiSettings } from 'react-icons/fi';
+import { AiFillRightCircle } from 'react-icons/ai';
+import { fetchApiDataTours } from '../redux/tours/toursAPI';
+import { PostReservationsAPI } from '../redux/reservations/reservations';
+
+import '../styles/reservation.css';
+
+const Reservation = () => {
+  const userStore = useSelector((store) => store.user);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (userStore.length < 1) {
+      navigate('/login');
+    }
+  }, [navigate, userStore]);
+
+  const [value, setValue] = useState({
+    date: '',
+    tour_id: '',
+  });
+
+  const onChange = (e) => {
+    setValue({ ...value, [e.target.id]: e.target.value });
+    console.log(value);
```

Comment:
- [ ]  Please, kindly remove this old piece of debugging code from the repo (console.log).  That way, it will be cleaner.  💪